app/src/main/index.js

```diff
@@ -90,15 +91,22 @@ function shutdown () {
   )
 }
 
+let appIcon = nativeImage.createFromPath(join(__dirname, '../icons/png/icon.png'))
+
 function createWindow () {
   mainWindow = new BrowserWindow({
     minWidth: 320,
     minHeight: 480,
     width: 1200,
     height: 800,
+    center: true,
+    title: 'Cosmos',
+    icon: appIcon,
+    darkTheme: true,
+    titleBarStyle: 'hiddenInset',
```

Comment:
@nylira this removes the grey bar at the top — but leaves the 3 buttons overlapping the logo...

Reply:
I *love* how this looks. There are some issues with this though:

1. The app menu on mobile is also blocked by the 3 buttons. We can resolve this by moving the app menu to the right hand side.
2. There's no longer any way to drag the app window around. @jolesbi can you investigate how to make the top 48px of our app draggable?
3. How does this look on Windows?

Reply:
Please add screenshots of what to compare to. ;) I can't see any change:
![image](https://user-images.githubusercontent.com/5869273/33479945-4d2126c6-d68f-11e7-84b5-2ed12200e525.png)